.github/workflows/sanity-checks.yml

```diff
@@ -1,6 +1,6 @@
 on:
-  # push:
-  #   branches: [main, 'release-v**']
+  push:
+    branches: ['ci-rewrite-n-cache']
```

Comment:
This should be changed back, right?

Reply:
Yes, it there was there just to trigger the CI in this PR